### DISCUSSION TITLE
IR sensor and IR led

### DISCUSSION THREAD

**Discussion by thiagovianav** [micropython/micropython#9190] [Hardware & Peripherals] [answered]
How I works with this in pico?

**Comment by jimmo** [comment, marked as answer]
(Moving this to the "Boards & Hardware" category. The "Core Development" is about working on MicroPython itself).

See https://github.com/peterhinch/micropython_ir

**Reply by thiagovianav** [reply]
But its How i work in micropython with this bro

**Reply by jimmo** [reply]
There are fairly detailed [Receiver docs](https://github.com/peterhinch/micropython_ir/blob/master/RECEIVER.md) and [Transmitter docs](https://github.com/peterhinch/micropython_ir/blob/master/TRANSMITTER.md) linked from that page.

**Reply by thiagovianav** [reply]
> There are fairly detailed Receiver docs and Transmitter docs linked from that page.

Oh right, Thanks for the help
